templates/projects/answerfieldsconfig.html

```diff
@@ -1,36 +1,25 @@
 {% extends "projects/base.html" %}
 {% set active_page = "projects" %}
 {% set active_link  = "settings" %}
-{% set section = _('Answer Fields Configuration') %}
+{% set section = _('Configuration') %}
 {% from "_formhelpers.html" import render_form %}
 
 {% block projectcontent %}
 
 <div id="answerfieldsconfig">
-    <div class="panel panel-danger">
-        <div class="panel-heading">
-            Warning
+    <div class="row">
+        <div class="col-md-5">
+            <h3> Consensus Configuration </h3>
+            <consensus-config :consensus_config="{{consensus_config}}"></consensus-config>
```

Comment:
Not a big deal, but it seems most of the JS uses camel case, e.g. `consensusConfig`.  It might be more standard to use a prop with name `:consensus-config` (which I think) will be passed to the `props` as `consensusConfig`.

Reply:
Reason that I named it with '_' is I wanted to distinguish with consensusConfig variable in store.js

Reply:
Is consensusConfig needed in the store?

Reply:
I think this should be camel case as Jaret mentioned, also the lint shows it as a warning. Vue automatically allow you to used it as :consensus-config